### PR TITLE
17088-update wording.

### PR DESCRIPTION
**Pull request by mengdong19**
*Issue #:* /bcgov/entity#17088

*Description of changes:*
Update wording.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts
Comment (severinbeauvais) [inline src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts:39]: Can you please check with UX team on whether to use "BC" or "B.C."? I see conflicting requirements between various tickets/uxpins.

File: package-lock.json
Comment (severinbeauvais) [reply package-lock.json]: Thanks for making this version 2.

@ozamani9gh @eve-git FYI

Comment (severinbeauvais) [reply package-lock.json]: I'm not sure why the lockfile has a conflict, but it will go away if you do `npm i` (and then commit the new file). Thanks.

**Comment by severinbeauvais** [comment]
Please get another review or 2 before you can merge this.

File: src/main.ts
Comment (severinbeauvais) [inline src/main.ts:116]: I'm surprised to see this here, since it is bolyachevets' commit from last week and it's [already in main branch](https://github.com/bcgov/namerequest/blob/93f82f43983165d8661024fb50ebe4d56ddd9343/src/main.ts#L109).

Anyway, I think there's no harm being done here.

**Review by JazzarKarim — APPROVED** [review]
LGTM! 👍

File: src/main.ts
Comment (mengdong19) [reply src/main.ts:116]: This PR was created before boylachevets's commit. I rebased. Looks like it's still comparing with the version when I created the PR.

File: src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts
Comment (severinbeauvais) [reply src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts:39]: Meng, you can merge after the above is decided/fixed, or you can merge now and fix the above in a second PR later.

File: src/main.ts
Comment (severinbeauvais) [reply src/main.ts:116]: Yeah, weird. I think I've seen this before in Github side-by-side compares. Anyway, not a problem here. Resolving.

File: src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts
Comment (mengdong19) [reply src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts:39]: Name request is using "BC", please see my comments in original ticket, I'll update after UX team decide what to use.

**Pull request merged**
